src/ui/shared/Modal.tsx

```diff
@@ -52,14 +52,14 @@ export default function Modal({
                   <Button
                     onClick={backButton.action}
                     label={backButton.label}
-                    className="px-4 py-2 text-lg font-medium text-white bg-gray-600 rounded-full hover:bg-gray-500 focus:outline-none focus:ring-2 focus:ring-gray-400 w-36 text-nowrap"
+                    className="px-4 py-2 text-lg font-medium text-white bg-gray-600 rounded-full focus:outline-none focus:ring-2 focus:ring-gray-400 w-36 text-nowrap"
                   />
                 )}
                 {forwardButton && (
                   <Button
                     onClick={forwardButton.action}
                     label={forwardButton.label}
-                    className="px-4 py-2 text-lg font-medium text-white bg-twd-primary-purple rounded-full hover:bg-purple-600 focus:outline-none focus:ring-2 focus:ring-purple-400 w-36 text-nowrap"
+                    className="px-4 py-2 text-lg font-medium text-white bg-twd-primary-purple rounded-full focus:outline-none focus:ring-2 focus:ring-purple-400 w-36 text-nowrap"
```

Comment:
Another one [anotherone](https://media0.giphy.com/media/3o7WTxyMSVN7lM5I7C/giphy.gif?cid=6c09b952htsv5wzqd6cu2k5wrmvbbqtr2zp0gdvpiwti5nha&ep=v1_internal_gif_by_id&rid=giphy.gif&ct=g)

Reply:
without it the browser/device overrides the styling, so our purple outline isn't visible. As far as I'm aware, the fact that we're removing it but adding our own custom outline with focus:ring should mean that this element meets accessibility requirements